src/com/owncloud/android/ui/adapter/FileListListAdapter.java

```diff
@@ -326,6 +326,8 @@ public View getView(int position, View convertView, ViewGroup parent) {
                         Drawable backrepeat = mContext.getResources().
                                               getDrawable(R.drawable.backrepeat);
                         fileIcon.setBackground(backrepeat);
+                    } else {
+                        fileIcon.setBackground(null);

```

Comment:
Is this line necessary for the fix?

I think this line is from another fix (https://github.com/owncloud/android/pull/1113 and https://github.com/owncloud/android/pull/1114). Besides at the end this line had been removed in this fix.

Maybe, if you update this PR with the last version of master branch this line will disappear.